bookkeeper-server/conf/bk_server.conf

```diff
@@ -40,11 +40,21 @@ bookiePort=3181
 # establish their identities as 127.0.0.1:3181, and only one will be able
 # to join the cluster. For VPSs configured like this, you should explicitly
 # set the listening interface.
-#allowLoopback=false
+# allowLoopback=false
+
+# Whether the bookie allowed to use an ephemeral port (port 0) as its server port.
+# By default, an ephemeral port is not allowed. Using an ephemeral port
+# as the service port usually indicates a configuration error. However, in unit
+# tests, using ephemeral port will address port conflicts problem and allow
+# running tests in parallel.
+# allowEphemeralPorts=false
 
 # Directory Bookkeeper outputs its write ahead log
 journalDirectory=/tmp/bk-txn
 
+# Directories Bookkeeper outputs its write ahead log
```

Comment:
Directories BookKeeper outputs its write ahead log.
Could define multi directories to store write head logs, separated by ','.
For example:
journalDirectories=/tmp/bk-journal1,/tmp/bk-journal2

if `journalDirectories` is set, bookies will skip `journalDirectory` and use this setting directory.